dev/system-test/firestore.ts

```diff
@@ -2265,6 +2275,27 @@ describe('QuerySnapshot class', () => {
   });
 });
 
+describe('BulkWriter class', () => {
+  let firestore: Firestore;
+  let randomCol: CollectionReference;
+
+  beforeEach(() => {
+    firestore = new Firestore({});
+    randomCol = getTestRoot(firestore);
+  });
+
+  afterEach(() => verifyInstance(firestore));
+
+  it.skip('can terminate once BulkWriter is closed', async () => {
```

Comment:
Add TODO(bulkwriter) or something enable this test>

Reply:
done.